templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -3,12 +3,13 @@
 ![Mention a Team Member](/static/images/help/mention_user.png)
 
 When having a conversation in Zulip, it is sometimes necessary to call
-the attention of another member. You can alert ("**@mention**") other
-users with a particular stream-message from within the body of the
-message itself. Type `@` and then begin typing their email address or
-one of their names, and the system will offer you tab-completions. The
-completed `@mention` will look something like: `@**name**`, and it
-will appear as **@name** in your posted message.
+the attention of another member. You can alert other
+users within a particular stream through using **@mention** in the in
+the **Compose Message** box. Type `@` followed by their email address or one of
+their names (first name or surname), and
+Zulip will offer you tab-completions. A completed `@mention` will look
+something like: `@**name**`, where **name** is the user you want to alert, and it
+will appear as @\*\*name\*\* in the markdown of your message.
 
```

Comment:
`templates/zerver/help/index.md` has a note on this doc: "(needs a note that you can't @mention when editing messages, since they may have already read the message / not clear how to notif them)" Please modify your doc to fix the changes requested in `index.md`